cfgov/jobmanager/tests/test_views.py

```diff
@@ -0,0 +1,54 @@
+import django
+
+from django.test import RequestFactory
+from mock import patch
+from model_mommy import mommy
+from unittest import TestCase
+
+from jobmanager.models import Job
+from jobmanager.views import CurrentOpeningsView, IndexView, JobListView
+
+
+django.setup()
```

Comment:
This call (and using `unittest.TestCase` instead of `django.test.TestCase`) allows you to run unit tests quickly without depending on the database, e.g.:

```sh
DJANGO_SETTINGS_MODULE=cfgov.settings.test python -m unittest jobmanager.tests.test_views
```